src/containers/Form/FormHeader.jsx

```diff
@@ -49,53 +69,216 @@ const FormHeader = props => {
     { key: 'unknown', title: t('language.unknown'), include: false },
     { key: 'de', title: t('language.de'), include: false },
   ];
-  const language = languages.find(lang => lang.key === model.language);
-  if (!model.id) {
-    return (
-      <div {...formClasses('header', types[type].cssModifier)}>
-        <div>
-          {types[type].icon}
-          <span>{t(`${types[type].form}.title`, language)}</span>
-        </div>
-      </div>
-    );
-  }
 
   const emptyLanguages = languages.filter(
     lang =>
       lang.key !== model.language &&
       !model.supportedLanguages.includes(lang.key) &&
       lang.include,
   );
-  const otherLanguages = model.supportedLanguages.filter(
-    lang => lang !== language.key,
+
+  const newLanguage =
+    model.id && !model.supportedLanguages.includes(model.language);
+  const isNew = newLanguage && (
+    <div css={statusWrapperCSS}>
+      <div css={splitterCSS} />
+      <p css={statusCSS}>{t('form.status.new_language')}</p>
+    </div>
   );
 
   return (
-    <div {...formClasses('header', types[type].cssModifier)}>
-      <div>
-        {types[type].icon}
-        <span>{t(`${types[type].form}.title`, language)}</span>
-        {otherLanguages.map((lang, index) => (
-          <span key={`types_${lang}`}>
-            <Link to={editUrl(lang)}>{lang.toUpperCase()}</Link>
-            {index === otherLanguages.length - 1 ? '' : '|'}
-          </span>
-        ))}
+    <header>
```

Comment:
En ganske stor render med mye sjekker for om ting skal vises. Er det mulig å splitte opp, og gjøre den mer lesbar?